mne/viz/backends/renderer.py

```diff
@@ -59,17 +59,15 @@ def set_3d_backend(backend_name):
        +--------------------------------------+--------+---------+
        | 3D function:                         | mayavi | pyvista |
        +======================================+========+=========+
-       | :func:`plot_source_estimates`        | ✓      |         |
+       | :func:`plot_source_estimates`        | ✓      | -       |
        +--------------------------------------+--------+---------+
        | :func:`plot_alignment`               | ✓      | ✓       |
        +--------------------------------------+--------+---------+
        | :func:`plot_sparse_source_estimates` | ✓      | ✓       |
        +--------------------------------------+--------+---------+
        | :func:`plot_evoked_field`            | ✓      | ✓       |
        +--------------------------------------+--------+---------+
-       | :func:`snapshot_brain_montage`       | ✓      | ✓       |
-       +--------------------------------------+--------+---------+
-       | :func:`plot_evoked_field`            | ✓      |         |
```

Comment:
What happened to `plot_evoked_field`?

Reply:
It appeared twice on the table for some reason, I just correct it in here.